icubam/predicu/tests/data/bedcounts.csv

```diff
@@ -0,0 +1,1054 @@
+icu_name,date,datetime,department,n_covid_deaths,n_covid_healed,n_covid_transfered,n_covid_refused,n_covid_free,n_ncovid_free,n_covid_occ,n_ncovid_occ
```

Comment:
I am very scared by these .csv files, because just by looking at it it's hard to know that this is not senstivite production data. If necessary, please add a comment on the top to indicate that we would want to remove it in the future in favor of using DB, and rename them to clearly indicate that it's only test data;

Reply:
Also let's add a `*.csv` to `.gitignore` to make it harder to commit sensitive csv by accident.

Reply:
i believe them being located in `icubam/predicu/tests/data` makes it pretty clear already that this is test data. this data was obfuscated by way of randomisation. it contains random numbers.

Reply:
> i believe them being located in icubam/predicu/tests/data makes it pretty clear already that this is test data.

Great! To me it wasn't obvious. If it contains random numbers, then let's add this with a comment on the top,
```
# This file contains a randomly generated dataset for testing
```

Reply:
pandas did not accept this comment

Reply:
Need to pass `comment="#"` to pd.read_csv..

Reply:
thanks, done.